index.test.js

```diff
@@ -94,13 +121,14 @@ describe('Render task definition', () => {
         expect(core.setOutput).toHaveBeenNthCalledWith(1, 'task-definition', 'new-task-def-file-name');
     });
 
-    test('renders a task definition at an absolute path, and with initial environment empty', async () => {
+    test('renders a task definition at an absolute path, and with initial environment and secrets empty', async () => {
```

Comment:
__nit:__ I might have understood this wrong and I see that the naming was always this way but the test name here is not accurate as the secrets and environment provided is not empty?